CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [24.0.2] - 2019-08-12
```

Comment:
25.0.0; we should mention the functions removed.

Reply:
right, will fix now